enterprise/server/workflow/service/service.go

```diff
@@ -261,16 +261,21 @@ func (ws *workflowService) GetWorkflows(ctx context.Context, req *wfpb.GetWorkfl
 	if err := ws.checkPreconditions(ctx); err != nil {
 		return nil, err
 	}
-
+	groupID, err := perms.AuthenticateSelectedGroupID(ctx, ws.env)
```

Comment:
Why read the proto RequestContext from ctx instead of from the request?

(I want to be v careful not to introduce code like this in internal libraries -- because we won't always have a proto RequestContext set depending on where calls originate.)

Reply:
Done. I don't remember exactly, but originally I think we might've made RequestContext part of the Context because all of our existing auth code was working with Contexts and taking in protos everywhere was probably going to be a lot of refactoring.

I'm fine with explicitly requiring the request to be passed in -- down the road if we ever want to refactor our auth code to require the proto message explicitly as well (instead of pulling it from ctx), that seems fine too.